webapp/cypress/integration/createBoard.ts

```diff
@@ -112,4 +112,45 @@ describe('Create and delete board / card', () => {
         cy.get('.DeleteBoardDialog button.danger').click({force: true})
         cy.contains(boardTitle).should('not.exist')
     })
+
+    it('scrolls the kanban board when dragging card to edge', () => {
```

Comment:
@venarius Requesting change below to include new test case identifier.
```suggestion
    it('MM-T4433 Scrolls the kanban board when dragging card to edge', () => {
```

@ogi-m Please help check the test case I've created related to this feature - [MM-T4433](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin:com.kanoah.test-manager__main-project-page#!/testCase/MM-T4433). Thanks in advance!